configure.ac

```diff
@@ -17,7 +17,7 @@ ERFA_NUMVER
 ## A library supports interfaces from current downto current - age
 ## Revision is the version of the current interface
 ## Follow the instructions in RELEASE.rst to change the version info
-ERFA_LIB_VERSION_INFO(2, 1, 1)
+ERFA_LIB_VERSION_INFO(2, 2, 0)
```

Comment:
I stared at that document for 5 minutes trying to work out what it meant if a new function is added but otherwise there is no change. Doesn't the 3rd number reset once the primary version number bumps? I will use whatever library version number you want.